packages/protect/README.md

```diff
@@ -0,0 +1,11 @@
+# protect
```

Comment:
We should also update this to be a bit more descriptive. We may be able to copy over the existing readme

https://github.com/ForgeRock/forgerock-javascript-sdk/blob/develop/packages/ping-protect/README.md

and slightly modify it.